src/shared/types/index.js

```diff
@@ -148,7 +148,7 @@ export const incidentType = PropTypes.shape({
     priority: PropTypes.string,
   }),
   signal_id: PropTypes.string,
-  source: PropTypes.oneOfType([PropTypes.string, PropTypes.shape({})]),
+  source: PropTypes.oneOfType([PropTypes.string, PropTypes.object]),
```

Comment:
`PropTypes.object` Shouldn't be used; use either `objectOf` or `shape`.

Reply:
My fault, is actually a shape({id, label})